aws/resource_aws_lambda_event_source_mapping.go

```diff
@@ -61,6 +61,13 @@ func resourceAwsLambdaEventSourceMapping() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validation.IsRFC3339Time,
 			},
+			"topics": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				ForceNew: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Set:      schema.HashString,
```

Comment:
AFAIK we can skip this

Reply:
I'm never sure either. I'll leave so as not to have to re-run the long acceptance test 😄.